octobot_evaluators/api/inspection.py

```diff
@@ -49,10 +49,12 @@ def is_relevant_evaluator(evaluator_instance, relevant_evaluators) -> bool:
 
 def get_relevant_TAs_for_strategy(strategy, config, tentacles_setup_config) -> list:
     ta_classes_list = []
-    relevant_evaluators = strategy.get_required_evaluators(config)
+    relevant_evaluators = strategy.get_required_evaluators()
     for ta_eval_class in create_advanced_types_list(TAEvaluator, config):
         ta_eval_class_instance = ta_eval_class()
         ta_eval_class_instance.set_tentacles_setup_config(tentacles_setup_config)
+        # force enable at True to use relevant_evaluators
+        ta_eval_class_instance.enabled = True
```

Comment:
All TA tentacles enabled ?

Reply:
I updated the method parameters, it should be more clear. For now  this method is used to display the TA list for a strategy on the strategy optimizer page